src/ui/ui_render/views/ui_app.jade

```diff
@@ -109,5 +113,7 @@ block content
         .kibanaWelcomeLogo
     .kibanaWelcomeText
       | #{i18n('UI-WELCOME_MESSAGE')}
+    #kibanaLoader__error
```

Comment:
__suggestion:__ what if we use data attribute instead?
```jade
.kibanaWelcomeText(data-error-message='#{i18n("UI-WELCOME_ERROR")}')
```

It won't require additional style with `display: none`, in js we can extract it like this:
```js
document.querySelector('[data-error-message]').dataset.errorMessage
// or `.getAttribute('data-error-message')` if we want to support IE 10 - trying to figure that out at the moment).
```

Or even maybe add this attribute to body in `chrome.jade` and then we won't need `querySelector` at all.

What do you think?

Reply:
@azasypkin makes some sense.  But still uses the i18n helper?

Reply:
> But still uses the i18n helper?

Yes, but see my comment above, we need Jade `i18n` helper, not the one we used in `Handlebars` template.

Reply:
Sorry, it's confusing, I know :)